botbot_plugins/plugins/github.py

```diff
@@ -67,9 +67,14 @@ def issue_lookup(self, line, repo_abbreviation, pulls):
         """Lookup an specified repo pulls"""
         # pulls can be a list of pulls separated by a comma
         pull_list = [i.strip() for i in pulls.split(",")]
+        # Check first in our stored values, allows overwriting hardcoded project_abbreviations values
         repo = self.retrieve(repo_abbreviation)
         if not repo:
-            return
+            if repo_abbreviation in project_abbreviations:
+                # Fallback to our hardcoded project_abbreviations
+                repo = project_abbreviations[repo_abbreviation]
```

Comment:
a "pythonic" way of doing this is `repo = project_abbreviations.get(repo_abbreviation)`.
`.get` will return `None` if the item doesn't exist (you can also specify a default if you want - `.get(repo_abbreviation, "placeholder")`, and it'll return that instead